js/atree_constants.js

```diff
@@ -1812,7 +1812,7 @@ const atrees = {
                 "slider_max": 4,
                 "output": {
                     "type": "stat",
-                    "name": "damMult.Basaltic:Basaltic Trap"
+                    "name": "damMult.Basaltic:7.Trap Damage"
```

Comment:
convention: `damMult.<unique_id>:<base_spell>.<part>`

ferri will kill me for this